crates/ty_python_semantic/src/semantic_model.rs

```diff
@@ -58,26 +58,44 @@ impl<'db> SemanticModel<'db> {
                 return vec![];
             }
         };
-        let Some(module) = resolve_module(self.db, &module_name) else {
+        self.module_completions(&module_name)
+    }
+
+    /// Returns completions for symbols available in the given module as if
+    /// it were imported by this model's `File`.
+    fn module_completions(&self, module_name: &ModuleName) -> Vec<Completion> {
+        let Some(module) = resolve_module(self.db, module_name) else {
             tracing::debug!("Could not resolve module from `{module_name:?}`");
             return vec![];
         };
         let ty = Type::module_literal(self.db, self.file, &module);
-        crate::types::all_members(self.db, ty).into_iter().collect()
+        crate::types::all_members(self.db, ty)
+            .into_iter()
+            .map(|name| Completion {
+                name,
+                builtin: module_name.as_str() == "builtins",
```

Comment:
nit: we usually use this API for checking whether a module is a specific module from a given search path with a given name

```suggestion
                builtin: module.is_known(KnownModule::Builtins),
```